packages/grid/src/vaadin-grid-selection-column-base-mixin.js

```diff
@@ -209,18 +209,6 @@ export const GridSelectionColumnBaseMixin = (superClass) =>
       }
     }
 
-    /** @private */
```

Comment:
This is needed to handle selection/deselection when dragging over a single checkbox. If not present, when dragging inside the checkbox area nothing happens.

Reply:
Thanks, I just noticed. There was a related code-structuring issue with preventing the active item from changing due to the click after drag end, which was implemented in the column element rather than in the mixin. That is not going to work as such for the Flow component, which does not use the column element itself, and has its own active item selection logic.

So I restored the click handler and made it work so that it prevents both the checkbox click and toggling the active item: https://github.com/vaadin/web-components/pull/6098/commits/6927532aae4a550ad8b9c7926a558854248f0179